client/elife-theme/src/index.js

```diff
@@ -44,16 +44,7 @@ export default {
 
   /* Spacing */
   gridUnit: `${gridUnit}px`,
-  space: [
-    0,
-    gridUnit, // XXS
-    gridUnit * 2, // XS
-    gridUnit * 4, // S
-    gridUnit * 6, // M
-    gridUnit * 8, // L
-    gridUnit * 12, // XL
-    gridUnit * 20, // XXL
-  ],
+  space: [0, 1, 2, 4, 6, 8, 12, 20].map(factor => `${factor * gridUnit}px`),
```

Comment:
Isn't it annoying to have to do the math every time we need to figure out what index to use?

Reply:
Yes it is. Put the comments back (and maybe add the actual pixel sizes too since XL doesn't mean much either).

Reply:
Maybe a comment like:
```
// space.6 = 72px 
```
... that way searching will also reveal it ;)

Reply:
Seems like it defeats the point of using a variable in the first place, no?

If the values change in the theme, you'd then need to update each comment that exists next to wherevver `space.6` is used :disappointed: 

The only reason we need the `space` array, is for `grid-styled`, right? How about we leave the space array in the theme, but let it be used for that purpose only?

Everywhere else in our code, we calculate explicitly i.e. `gridUnit * 12`. You still don't have anywhere that explicitly says "this is 72px" :woman_shrugging:  But it's easy enough to keep one variable in your head (gridUnit = 6px) & do multiplications like this in your head: 6px x 12.

Whereas it's really hard trying to remember what `space.6` refers to. After all `6` is just the 6th position in a list & that list is generated based on a fairly arbitrary scale.

Whaddaya reckon?